scripts/kconfig/hardened.csv

```diff
@@ -1,6 +1,5 @@
 HW_STACK_PROTECTION,y
```

Comment:
would be nice to add a comment at the top telling whoever modifies this file in the future to keep the list sorted alphabetically.

Reply:
+1, but this is making me realize that the file supposedly being "CSV", I am not so sure whether we want comments in the first place. I know that I would never thought of adding a comment such as https://github.com/zephyrproject-rtos/zephyr/blob/bfdfbd92d6854da8273826ace1fa8ca9ae81eb68/scripts/kconfig/hardened.csv#L49 myself, for the very reason that I would not expect it to work. It turns out that this probably ends up caught [here](https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/kconfig/hardenconfig.py#L51-L54) but this feels hackish, no?
But don't get me wrong, I definitely like the conciseness of a CSV-like syntax in this particular instance.

Reply:
Regarding comments, it is catch in that exception, it is not different from an empty line. I can add some logic to explicitly handle it but it seems an unnecessary overhead.

I put these comments to separate some logical sections, I think it makes it easier to review it.